frontend/server/src/Validators.php

```diff
@@ -61,6 +61,19 @@ public static function validateStringNonEmpty(
         }
     }
 
+    /**
+     * Check whether parameter value is string and not empty
+     *
+     * @param mixed $parameter
+     * @psalm-assert string $parameter
+     */
+    public static function stringNonEmpty($parameter): bool {
```

Comment:
```suggestion
    public static function stringNonEmpty(string $parameter): bool {
```

aquí sí sabemos que es una cadena.

Reply:
Listo